crypto_box/src/lib.rs

```diff
@@ -94,7 +94,7 @@
 //! assert_eq!(&plaintext[..], &decrypted_plaintext[..]);
 //! ```
 //!
-//! ## Customized Associated Data Usage
+//! ## Choosing XChaCha20Poly1305 vs XSalsa20Poly1305
```

Comment:
Thanks for changing this! I had a second thought though: it should probably be:

```suggestion
//! ## Choosing `ChaChaBox` vs `SalsaBox`
```

Sorry about that 😅

Reply:
haha right, you are right, that is clearer, no worry, updating now.

Reply:
do you want me to add "`SalsaBox(default)` ?

Reply:
Nah, the `Box` type alias makes that clear, I think